Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageListView.swift

```diff
@@ -230,8 +230,8 @@ public struct MessageListView<Factory: ViewFactory>: View, KeyboardReadable {
                             } else if diff < 0 && scrollDirection == .down {
                                 scrollDirection = .up
                             }
+                            utils.messageCachingUtils.scrollOffset = offsetValue
```

Comment:
Here we had an issue where if you scroll extra slowly, the scroll direction argument in the view model is incorrect.